src/style/search/search-bar.less

```diff
@@ -67,10 +71,17 @@
 .search-bar__dialog .drop-down .hiddeninput {
     display: inline-block;
     position: absolute;
-    left: 3.2rem;
+    margin-left: .1rem;
     visibility: hidden;
     white-space: pre;
 }
+.search-bar__dialog .drop-down .hiddenSVG {
```

Comment:
The naming on some of these classes is not consistent with the rules in our style guide here: https://github.com/adobe-photoshop/spaces-design/wiki/Coding-Conventions#naming-and-syntax